src/options.json

```diff
@@ -0,0 +1,61 @@
+{
+  "type": "object",
+  "properties": {
+    "config": {
+      "type": "object",
+      "properties": {
+        "path": {
+          "type": "string"
+        },
+        "ctx": {
+          "type": "object"
+        }
+      },
+      "errorMessage": {
+        "properties": {
+          "ctx": "should be {Object} (https://github.com/postcss/postcss-loader#context-ctx)",
+          "path": "should be {String} (https://github.com/postcss/postcss-loader#path)"
+        }
+      },
+      "additionalProperties": false
+    },
+    "exec": {
+      "type": "boolean"
+    },
+    "ident": {
```

Comment:
`ident` is property for webpack, we should not check this here, this do webpack already

Reply:
Since we can't change `addtionalProperties` to `false` due to third parties anyways I will test and remove it